### PR TITLE
Fix CheckBoxGroup defaults, add test

### LINKED ISSUE #1268: Bug - Restore default values for Form example
The [forms filter example](https://design-system.hpe.design/templates/forms#filter) previously initialized with values for some checkboxes checked. This was lost when the example was refactored to use `<CheckBoxGroup />`.

- [x] Confirm expected syntax for setting CheckBoxGroup values in form value state
- [x] Confirm FormContext and CheckBoxGroup are working as expected
- [x] Should a test or story be added/modified for CheckBoxGroup in a controlled form?

### PR DESCRIPTION
**Pull request by taysea**
#### What does this PR do?
Restores default values for CheckBoxGroup. No grommet changes necessary, just needed to align format of form value defaults to CheckBoxGroup format. Added a snapshot test for entire forms page to capture any default values we have defined. Cleaned up CheckBoxScroll example to use "overflow=auto" instead of "scroll" so that scroll bars only present if necessary.

#### Where should the reviewer start?
aries-site/src/examples/templates/forms/FilterExample.js

#### What testing has been done on this PR?
Tested locally.

#### How should this be manually tested?
Check deploy.

#### What are the relevant issues?
Closes #1268 

#### Should this PR be mentioned in Design System updates?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes, backwards compatible.

### CODE REVIEW

File: aries-site/src/examples/templates/forms/FilterExample.js [hunk 1]
```diff
@@ -33,10 +33,8 @@ export const FilterExample = () => {
   const [formValues, setFormValues] = React.useState({
     keyword: '',
     locationType: 'All Locations',
-    'Rack Servers': true,
-    'Tower Servers': true,
-    Amazon: true,
-    Insight: true,
+    'server-types': ['Rack Servers', 'Tower Servers'],
+    sellers: ['Amazon', 'Insight'],
```
Comment (halocline) [inline aries-site/src/examples/templates/forms/FilterExample.js:37]: 👍

File: aries-site/src/examples/templates/forms/FilterExample.js [hunk 2]
```diff
@@ -85,13 +83,21 @@ export const FilterExample = () => {
             />
           </FormField>
           {serverTypes && (
-            <FormField label="HPE Server Types" htmlFor="server-types">
-              <CheckBoxGroup options={serverTypes} id="server-types" />
+            <FormField
+              label="HPE Server Types"
+              htmlFor="server-types"
+              name="server-types"
```
Comment (halocline) [inline aries-site/src/examples/templates/forms/FilterExample.js:89]: 👍

**Review by halocline — APPROVED** [review]
Looks good.

Visual snapshot tests have not yet completed.

**Comment by taysea** [comment]
> Looks good.
> 
> Visual snapshot tests have not yet completed.

Sweet. I will have to approve the new snapshots for the forms page, but after that it should be good to go.

**Pull request merged**